com.unity.multiplayer.mlapi/Runtime/Messaging/RpcParams.cs

```diff
@@ -1,13 +1,34 @@
 namespace MLAPI.Messaging
 {
-    public struct ServerRpcSendParams
+    public interface IHasUpdateStage
     {
-        public NetworkUpdateStage UpdateStage;
+        public NetworkUpdateStage UpdateStage
+        {
+            get;
+            set;
```

Comment:
Maybe consider making the interface itself get only so that we can create readonly structs. Looks like the two cases where this is used in RPC params should be readonly?

Reply:
In this case I don't think it should be readonly for two reasons.

First, when the value is "Unset", it has to be able to be changed by the RPC sending code.

Second, this value is input from our users. At least on the SendParams, anyway. I think it would be odd and unintuitive to give players a struct they construct themselves and say "you can provide the update stage when you create it, but you can't change it afterward"

Reply:
Actually, the whole reason I made this interface was to be able to change that value in a generic function. So making it readonly would defeat the purpose of making the interface. :)